tools/teal/templates/limit-order.teal.tmpl

```diff
@@ -45,9 +68,9 @@ gtxn 1 AssetAmount
 &&
 &&
 int 1
-bnz label7
+bnz label3
 pop // check bug workaround
```

Comment:
Do we still need that?

Reply:
Ah, this PR does not remove the workaround. The PR that fixes that is here: 

https://github.com/algorand/go-algorand/pull/481

I'm thinking this PR can be merged independently from that one.

Reply:
👍